File: Session_01_BasicPagramming/Lab01/Python_Lang/Demo12.py
#
#  Created by NDHuy at 12/04/2020
#  Copyright 2020 by SEstudio
#  Check perfect square number
#

def main():
    n = int(input("Enter a iteger number n > 0: "))

    if n <= 0:
        print("%d is not greater than Zero" %(n))
    else:
        flag = False
        if (n == 1):
            flag = True
        else:
            i = 1
            while i <= (n / 2):
                if (i * i) == n:
                    flag = True
                    break
                i += 1

        if (flag == True):
            print("%d is a perfect square number" %(n))
        else:
            print("%d is not a perfect square number" %(n))


if __name__ == "__main__":
    main()

File: Session_01_BasicPagramming/Lab01/C_Lang/src/Demo12.cpp
//
//  Created by NDHuy at 12/04/2020
//  Copyright 2020 by SEstudio
//  Check perfect square number
//

#include <stdio.h>

int main()
{
    int n; 
    printf("Enter a integer n > 0: ");
    scanf("%d", &n);

    if (n <= 0)
        printf("%d is not greater than Zero", n);
    else
    {
        if (n == 1)
        {
            printf("%d is a perfect square number\n", n);
        }
        else
        {
            int i;
            for(i = 1; i <= (n / 2); i++)
            {
                if (i * i == n) 
                {
                    printf("%d is a perfect square number\n", n);
                    break;
                }
            }

            if (i >= n/2)
                printf("%d is NOT a perfect square number\n", n);
        }
    }

    return 0;
}